### PR TITLE
[fix/github-translation-action-2] Github Translation Action Fix

### PR DESCRIPTION
**Pull request by hosy**
- removed wrong git path
- removed push trigger

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] Added an issue with details about all relevant changes in the [**iOS documentation repository**](https://github.com/owncloud/docs-client-ios-app/issues).
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Added changelog files for the fixed issues in folder changelog/unreleased

### CODE REVIEW

File: .github/workflows/translate.yml [hunk 1]
```diff
@@ -74,13 +71,11 @@ jobs:
           Set-Content -Value "${{ secrets.DEPLOYMENT_SSH_KEY }}" -Path ~/.ssh/id_ed25519
           chmod 0600 ~/.ssh/id_ed25519
           if(git stash list) {
-              git stash pop
               install -d -m 0700 ~/.ssh
               Set-Content -Value "${{ secrets.DEPLOYMENT_SSH_KEY }}" -Path ~/.ssh/id_ed25519
               chmod 0600 ~/.ssh/id_ed25519
               git config user.name "ownClouders"
               git config user.email "devops@owncloud.com"
-              git add translations/
```
Comment (TheOneRing) [inline .github/workflows/translate.yml:83]: Without those there will be nothing to commit.
Please fix the add step and keep the stash pop

Comment (hosy) [reply .github/workflows/translate.yml:83]: Pushed my changes, please have a look at the new commit

**Review by TheOneRing — APPROVED** [review]
Looks good thx

**Review by TheOneRing — APPROVED** [review]

**Comment by hosy** [comment]
@TheOneRing I did another commit to add all `strings` files in subdirectories

**Pull request merged**